packages/javascript-sdk/src/token-storage/index.ts

```diff
@@ -24,20 +23,23 @@ abstract class TokenStorage {
   /**
    * Gets stored tokens.
    */
-  public static async get(): Promise<Tokens> {
+  public static async get(): Promise<Tokens | void> {
     const { clientId, tokenStore } = this.getClientConfig();
 
     if (tokenStore === 'sessionStorage') {
       return await SessionStorageWrapper.get(clientId);
     } else if (tokenStore === 'localStorage') {
       return await LocalStorageWrapper.get(clientId);
+
+      // Preserving this condition for communicating its removal
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Removing the `indexedDb` type from the union below makes this comment necessary. I don't want to remove this condition as I want devs to know that it has been removed, so it's totally non-functional and just communicative.